README.md

```diff
@@ -1,7 +1,7 @@
-![CF](assets/shield-32x32.png) Lab 01: Modular Patterns and Testing
-===
+![cf](https://i.imgur.com/7v5ASc8.png) lab-02-build-tools
+======
 
-[![Issues in Lab Help](https://badge.waffle.io/codefellows/seattle-javascript-401n1.svg?label=question&title=Lab%20Help)](https://waffle.io/codefellows/seattle-javascript-401n1) [![Issues in Bug](https://badge.waffle.io/codefellows/seattle-javascript-401n1.svg?label=bug&title=Bug)](https://waffle.io/codefellows/seattle-javascript-401n1) [![Build Status](https://travis-ci.org/codefellows-seattle-javascript-401n1/lab-01-modular-patterns.svg?branch=master)](https://travis-ci.org/codefellows-seattle-javascript-401n1/lab-01-modular-patterns)
+[![Stories in Ready](https://badge.waffle.io/codefellows/seattle-javascript-401n1.svg?label=ready&title=Ready)](http://waffle.io/codefellows/seattle-javascript-401n1) [![Build Status](https://travis-ci.org/codefellows-seattle-javascript-401n1/lab-02-build-tools.svg?branch=master)](https://travis-ci.org/codefellows-seattle-javascript-401n1/lab-02-build-tools)  
 
 ##To Submit this Assignment
```

Comment:
You Shouldnt modify the assignmet code /readmefiles because when it gets merged it will change the assinment